frontend/components/icons/Apple.tsx

```diff
@@ -1,10 +1,33 @@
 import React from "react";
 
-const Apple = () => {
+interface IAppleProps {
+  size?: "small" | "medium" | "large";
+  color?: "coreFleetBlack" | "coreVibrantBlue" | "uiFleetBlack75";
+}
+
+const FLEET_COLORS = {
+  coreFleetBlack: "#192147",
+  coreVibrantBlue: "#6a67fe",
+  uiFleetBlack75: "#515774",
+};
```

Comment:
can we use the values in `styles/var/colors.ts` instead of redefining them here?